examples/show_starfile.py

```diff
@@ -0,0 +1,82 @@
+import napari
+
+
+from grace.io import starfile, core
```

Comment:
Better to have an explicit import of `GraceFile`

```suggestion
from grace.io import starfile
from grace.io.core import GraceFile
```